guides/common/modules/ref_utilities-for-processing-log-information.adoc

```diff
@@ -1,37 +1,49 @@
-[id="Utilities_for_Collecting_Log_Information_{context}"]
-= Utilities for Collecting Log Information
+[id="Utilities_for_Processing_Log_Information_{context}"]
+= Utilities for Processing Log Information
 
 You can collect information from log files to troubleshoot {Project}.
 
 ifdef::foreman-el,katello,satellite[]
 sosreport::
 The `sosreport` command collects configuration and diagnostic information from a Linux system, such as the running Kernel version, loaded modules, running services, and system and service configuration files.
 This output is stored in a tar file located at `/var/tmp/__sosreport-XXX-20171002230919.tar.xz__`.
++
 For more information, run `sosreport --help` or see https://access.redhat.com/solutions/3592[_What is a sosreport and how can I create one?_].
 endif::[]
-
 ifdef::foreman-deb,orcharhino[]
 foreman-debug::
 The `foreman-debug` command collects configuration and log file data for {ProjectName}, its back-end services, and system information.
-This information is collected and written to a tar file.
-By default, the output tar file is located at `/tmp/__foreman-debug-xxx.tar.xz__`.
-
+This output is stored in a tar file.
+By default, the tar file is located at `/tmp/__foreman-debug-xxx.tar.xz__`.
++
 Additionally, the `foreman-debug` command exports tasks run during the last 60 days.
 By default, the output tar file is located at `/tmp/__task-export-xxx.tar.xz__`.
 If the file is missing, see the file `/tmp/task-export.log` to learn why task export was unsuccessful.
 There is no timeout when running this command.
 endif::[]
-
 ifdef::foreman-deb[]
++
 For more information, run `foreman-debug -h`.
 endif::[]
 ifdef::orcharhino[]
++
 For more information, run `orcharhino-debug -h`.
 endif::[]
-
++
 [IMPORTANT]
 ====
-The collection process removes security information such as passwords, tokens, and keys while collecting information.
+ifdef::foreman-el,katello,satellite[]
+The `sosreport` command
+endif::[]
+ifdef::foreman-deb,orcharhino[]
+The `foreman-debug` command
```

Comment:
Looking at line 30, Shouldn't this be `orcharhino-debug` for Orcharhino?

cc @maximiliankolb

Reply:
Thanks for the ping! Yes, the command is `orcharhino-debug`.

Reply:
Addressed.